packages/graphpack/config/webpack.config.js

```diff
@@ -47,6 +47,10 @@ module.exports = {
         test: /\.mjs$/,
         type: 'javascript/auto',
       },
+      {
+        test: /\.tsx?$/,
+        loader: 'ts-loader',
+      },
```

Comment:
It might be a little nicer to stay within the babel toolchain and instead rely on [`babel-preset-typescript`](https://babeljs.io/docs/en/babel-preset-typescript) + [`fork-ts-checker-webpack-plugin`](https://github.com/Realytics/fork-ts-checker-webpack-plugin).

Reply:
@damassi What’s the benefit of having the setup in Babel instead?

Reply:
Rather than having two compilers working in parallel, you have one, and can leverage everything that babel provides. It makes things a little more seamless in the end.

Reply:
I agree with @damassi 
Please check out the article https://iamturns.com/typescript-babel

Reply:
Makes sense, will switch, thanks!

Reply:
`babel-preset-typescript` is good, but does not support all TypeScript features (such as `const` enums,  `namespace`s and some more...)

https://babeljs.io/docs/en/babel-plugin-transform-typescript

Reply:
I think `namespace`s are in wide use, and the other features are not that critical.
For me, `tsc` works fine for most cases and servers, just need to combine it with the fork-checker and not use direct checking from Webpack.

Reply:
`namespaces` aren't in wide use due to ES6 modules. I saw a recent post from a Typescript product manager on this subject and he confirmed that the three features that aren't supported by babel 7 are essentially non issues. Let me try to dig that up.